### PR TITLE
Backport/8.15/pr 675

### PR DESCRIPTION
**Pull request by achuguy**
Backport https://github.com/elastic/rally-tracks/pull/675 to 8.15

### CODE REVIEW

File: elastic/security/templates/composable/logs-endpoint.events.file.json [hunk 1]
```diff
@@ -38,6 +36,9 @@
     "data_stream": {
       "hidden": false,
       "allow_custom_routing": false
+      {% if build_flavor != "serverless" %},
+      "failure_store": false
+      {% endif %}
```
Comment (gbanasiak) [inline elastic/security/templates/composable/logs-endpoint.events.file.json:41]: That's a snapshot build feature, so please remove.

Comment (achuguy) [reply elastic/security/templates/composable/logs-endpoint.events.file.json]: Done

**Comment by gbanasiak** [comment]
Waiting for https://github.com/elastic/rally-tracks/pull/700 to re-run the tests.

**Comment by gbanasiak** [comment]
@elasticmachine update branch

**Review by gbanasiak — APPROVED** [review]

**Pull request merged**